aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -197,23 +197,9 @@ private void MakeGrid()
             }
         }
 
-        public static void SwitchTile(Cell startTile)
+        public static void SetStartTilePropertyTrue(Cell startTile)
```

Comment:
Since this is a method that only gets called when the cell is the startTile, I would like to rename it to `UpdateSpawnPosition` or something like it. The current name is too descriptive. It suits the method now, but we may want to do other things when updating the spawn position.